recipe/pipeline.py

```diff
@@ -1,23 +1,23 @@
-# There are many comments in this example. Remove them when you've
-# finalized your pipeline.
+import argparse
+from fsspec.implementations.local import LocalFileSystem
+import os
 import pandas as pd
-import pangeo_forge
-import pangeo_forge.utils
-from pangeo_forge.tasks.http import download
-from pangeo_forge.tasks.xarray import combine_and_write
-from pangeo_forge.tasks.zarr import consolidate_metadata
-from prefect import Flow, Parameter, task, unmapped
-
-# We use Prefect to manage pipelines. In this pipeline we'll see
-# * Tasks: https://docs.prefect.io/core/concepts/tasks.html
-# * Flows: https://docs.prefect.io/core/concepts/flows.html
-# * Parameters: https://docs.prefect.io/core/concepts/parameters.html
-
-# A Task is one step in your pipeline. The `source_url` takes a day
-# like '2020-01-01' and returns the URL of the raw data.
-
-
-@task
+from pangeo_forge.recipe import NetCDFtoZarrSequentialRecipe
+from pangeo_forge.storage import CacheFSSpecTarget, FSSpecTarget
+from pangeo_forge.executors import PythonPipelineExecutor, PrefectPipelineExecutor
+import prefect
+from prefect import task, Flow
+import shutil
+import s3fs
+import tempfile
+import yaml
```

Comment:
What are we going with on linting/codestyles? AFAIK this isn't PEP8 compliant (which is being really picky, but I think we should just use something opinionated like `flake8`, `black` and `isort`)

[PEP8 imports](https://www.python.org/dev/peps/pep-0008/#imports)
```
Imports should be grouped in the following order:

Standard library imports.
Related third party imports.
Local application/library specific imports.

You should put a blank line between each group of imports.
```

Reply:
The recipe is linted when it goes through staged-recipes at https://github.com/pangeo-forge/staged-recipes/blob/0d6fa51a1189a7bdb1927e7987c7a74216d13ab2/.pre-commit-config.yaml.

I'd recommend copying that (or a similar) pre-commit config over when [creating a new repository](https://github.com/pangeo-forge/staged-recipes/blob/0d6fa51a1189a7bdb1927e7987c7a74216d13ab2/.github/workflows/create-repository.yaml).

Reply:
I'd argue linting then is a bit too late, I'd rather we upheld formatting across all our repos as they are, rather than transforming them when merged into others

It just makes everything look uniform then

Reply:
Just to clarify: recipes go through staged-recipes first, and *then* the GitHub repository is created for that recipe. So the linting will happen there first.

Reply:
Sure, that's fine, but if we're using this as an example, we should probably still lint/format this.

Reply:
Right:

> I'd recommend copying that (or a similar) pre-commit config over when creating a new repository.